Change from customized paranoid model to sequelize paranoid mode

UncleBu1728 opened the pull request:
changed `Group`, `User` and `GroupUser` from using the customized `ParanoidModel` to the sequelize `paranoid` setting

weihanwang commented on src/api/database/models/Group.ts:
```diff
@@ -25,12 +32,18 @@ import Role from "shared/Role";
 @Fix
-class Group extends ParanoidModel<
+class Group extends Model<
   InferAttributes<Group>,
   InferCreationAttributes<Group>
```
remove these two lines from all models

weihanwang commented on src/api/database/models/Group.ts:
```diff
@@ -25,12 +32,18 @@ import Role from "shared/Role";
+class Group extends Model<
   InferAttributes<Group>,
   InferCreationAttributes<Group>
   > {
```
Remove NonAttribute<> from everywhere

weihanwang replied:
I don't see this comment was addressed.

UncleBu1728 commented:
> changed `Group`, `User` and `GroupUser` from using the customized `ParanoidModel` to the sequelize `paranoid` setting

Removed `InferAttributes` in the previous commit
Removed `Nonattribute` branding in the latest commit 

The following tests are completed and passed
- Syncing database with an empty db √
- Inside the db tables
  - Adding and deleting rows √
  - Manipulating existing data √
  - Deleting data in the paranoid model √
  - Adding columns √
    - Do not drop columns with syncing
- Passed both on Mac and Windows systems

TODO: 
-fixing yarn `gen-test-data`
